logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionMapper.kt

```diff
@@ -10,12 +10,11 @@ import com.wire.kalium.logic.data.user.type.UserType
 import com.wire.kalium.logic.di.MapperProvider
 import com.wire.kalium.network.api.user.connection.ConnectionDTO
 import com.wire.kalium.persistence.dao.ConnectionEntity
-import com.wire.kalium.persistence.dao.UserEntity
 
 interface ConnectionMapper {
     fun fromApiToDao(state: ConnectionDTO): ConnectionEntity
-    fun fromDaoToModel(state: ConnectionEntity, otherUser: UserEntity?): Connection
-    fun fromDaoToConnectionDetails(state: ConnectionEntity, otherUser: UserEntity?): ConversationDetails
+    fun fromDaoToModel(connection: ConnectionEntity): Connection
+    fun fromDaoToConnectionDetails(connection: ConnectionEntity): ConversationDetails
```

Comment:
tiny thing but wouldn;t be connectionEntity better choice as we are having Connection domain object already ? 

Als toConversationDetails

Reply:
I just removed the userEntity from here! didn't want to make more changes :D